activities/Measure.activity/js/activity.js

```diff
@@ -770,11 +779,11 @@ var app = new Vue({
 						doc.text(x, y, this.SugarL10n.get("Session") + ": " + i);
 						doc.setFontStyle("normal");
 						doc.text(this.SugarL10n.get("Mode") + ":     " + session.mode, x, y + 14);
-						if (this.log_interval == 300) {
+						if (session.interval == "300 second") {
 							doc.text(this.SugarL10n.get("Interval") + ":   5" + " " + this.SugarL10n.get("Minutes_other"), x, y + 21);
 						}
 						else {
-							doc.text(this.SugarL10n.get("Interval") + ":   " + this.log_interval + ' ' + this.SugarL10n.get("Second"), x, y + 21);
+							doc.text(this.SugarL10n.get("Interval") + ":   " + session.interval.split(' ')[0] + ' ' + this.SugarL10n.get("Second"), x, y + 21);
```

Comment:
Will be better to set interval as a parameter of Second string (using {{interval}} syntax) because it some language the number could be postfixed.